src/plays/simple-live-chat/components/chat/chat-body.jsx

```diff
@@ -0,0 +1,132 @@
+import format from "date-fns/format";
+import { Fragment, useEffect, useRef, useState } from "react";
+
+// css
+import "./chat-body.scss";
+
+// assets
+import userIcon from "../../images/user_icon.png";
+
+const User = ({ photoURL, onClick }) => {
+  const [defaultImage, setDefaultImage] = useState(photoURL);
+  return (
+    <img
+      src={defaultImage}
+      alt='avatar'
+      onClick={onClick}
+      className='simple-live-chat-user'
+      onError={() => setDefaultImage(userIcon)}
+    />
+  );
+};
+
+const ChatBody = ({ message, uid, loading }) => {
+  const [node, setNode] = useState([]);
```

Comment:
a more specific naming would be better here. `node` being a well-known terminology, I would suggest putting something like `chatNode` or so.

Reply:
yes its changed now